dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -467,7 +466,7 @@ public List<T> filterTrsToolsGet(DescriptorLanguage descriptorLanguage, String r
 
         final HibernateCriteriaBuilder cb = currentSession().getCriteriaBuilder();
         final JpaCriteriaQuery<T> q = cb.createQuery(typeOfT);
-        final Root<T> tRoot = generatePredicate(descriptorLanguage, registry, organization, name, toolname, description, author, checker, cb, q);
```

Comment:
This will be removing functionality from this method, as it's now ignoring the `author` parameter from line 466/465.

Can you add the `author` parameter back to `generatePredicate`, and have it query against the new location for the author?

Reply:
I realized that my ticket description was not very clear on that part :sweat_smile:, but ditto on what Charles said - we should be querying from the new `authors` location instead

Reply:
Note for self:
>> The tricky part is that the criteria query  entry.get("author") on a line like [this](https://github.com/dockstore/dockstore/blob/12b51f184c6651d99dcb1028b587e6d2450585d4/dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java#L414) no longer exists once the author column is removed, so we need replace it with authors which is our new way of tracking authors. The issue is that entry.get("authors") doesn't work because authors is a set, which is stored in another table in the database. I can't seem to figure out how to get the list of authors for an entry using the [JPA Criteria Queries](https://www.baeldung.com/hibernate-criteria-queries).